### DISCUSSION TITLE
Third Party Sync Services

### DISCUSSION THREAD

**Discussion by jocmp** [jocmp/capyreader#281] [Feature Requests]
I'm looking for interest in other sync services beyond Feedbin. If you use a sync service, vote!

**Update**

FreshRSS and Google Reader API are now supported. See the announcement for more details 

- https://github.com/jocmp/capyreader/discussions/492

**Comment by jocmp** [comment]
Small update: I've searched high and low for Feedly's REST API. I believe they're moving their API away from RSS and into [thread intelligence only](https://developers.feedly.com/) 👎 Please let me know if there's support otherwise.

On the other hand, FreshRSS has a few different options, and I'd like to implement this within the coming months.

**Reply by Monirzadeh** [reply]
Fever API and Google Reader API usual y posible to use in different program so if capy reader can support them is it possible to use capy reader  for them

**Reply by jstride** [reply]
I believe the greader API offers something to easily mark read on scroll compared to frver (based on functionality in my ios reader)

**Reply by jocmp** [reply]
I haven't used Fever or the Google Reader API before. Based on what I've read most sync services that support Google Reader API also support Fever. It seems like gReader API is preferred?

**Reply by Monirzadeh** [reply]
It is depend on user reader back end
So in long term maybe both but short term google api.
What do you think?

**Reply by jocmp** [reply]
Yeah I'd like to add both to Capy Reader at some point, it's really just a matter of time and whether they get used - for instance if every service that offers GReader also offers Fever, it would be easier to maintain just one.

**Reply by jstride** [reply]
For what it's worth https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html has some background reading (including sync strategies at the bottom), it also links to the main README where it says greader is better/more efficient. That's the context for my prior comment.

**Reply by woj-tek** [reply]
> I haven't used Fever or the Google Reader API before. Based on what I've read most sync services that support Google Reader API also support Fever. It seems like gReader API is preferred?

For example miniflux supports both so support for either would be awesome :)

<img width="543" alt="Captura de pantalla 2024-11-05 a las 9 29 33" src="https://github.com/user-attachments/assets/cbaf584c-c47e-4368-a9e6-b36e8504e7a1">

**Reply by jocmp** [reply]
@woj-tek that's good to know! I didn't know Google Reader was an option for Miniflux. 

I'm making progress on adding FreshRSS. Behind the scenes the code supports any Google Reader server. I'm planning on adding an option on the "Add Account" screen for Google Reader in addition to FreshRSS specifically.

**Reply by woj-tek** [reply]
Awesome, that would cover lots of services IMHO as many (self hosted ones) support Google Reader API :)

**Comment by jocmp** [comment]
No timeline yet but I'm going to add FreshRSS first by popular demand. 

- https://github.com/jocmp/capyreader/issues/332

**Reply by jocmp** [reply]
First pass at FreshRSS support is now available https://github.com/jocmp/capyreader/discussions/492

**Reply by jstride** [reply]
I've been using it since it was released and it is working well so far, thanks @jocmp - one query - does Capy have mark as read on scroll?

**Reply by jocmp** [reply]
hey good to hear. and not yet, I'd like to add it but when I tried awhile back I wasn't happy with my implementation. I'll pull the ticket back onto the project board here

- https://github.com/jocmp/capyreader/issues/271

**Comment by valpackett** [comment]
I've been using [yarr](https://github.com/nkanaev/yarr/tree/master) which [implements Fever's API](https://github.com/nkanaev/yarr/blob/master/doc/fever.md) as the only API. Would be nice to have a good Android client. ReadYou [doesn't like uncategorized feeds](https://github.com/Ashinch/ReadYou/issues/867) in its Fever implementation, which is also seemingly deprecated :roll_eyes:

**Reply by jocmp** [reply]
I've noticed that both Reeder on macOS and ReadYou list Fever API as deprecated. I believe this is because the [underlying service Fever](https://web.archive.org/web/20230303040251/https://feedafever.com/) is no longer available. But the same is true for Google Reader after all!

I'll track Fever support via https://github.com/jocmp/capyreader/issues/421 with yarr in mind.

**Reply by valpackett** [reply]
> the underlying service Fever is no longer available

Note that it is self-hosted (PHP) software, not a service. Not officially maintained, but it never stopped working for those who have it :)

**Comment by Monirzadeh** [comment]
i use google google api to connect miniflux and it is work fine but i can't see any thumbnail for my feeds.
do i miss something?

**Reply by jocmp** [reply]
hey @Monirzadeh What feed is missing thumbnails for you? It might be possible that Miniflux isn't picking up the images. Capy looks for the enclosures returned by the Google Reader API. More details on that here: https://github.com/jocmp/capyreader/discussions/562#discussioncomment-11389888
